src/extension.js

```diff
@@ -35,7 +35,17 @@ async function getPullRequests(context, showError) {
 		const showMerged = vscode.workspace.getConfiguration('pullRequestMonitor').get('showMerged');
 		const showClosed = vscode.workspace.getConfiguration('pullRequestMonitor').get('showClosed');
 		const count = vscode.workspace.getConfiguration('pullRequestMonitor').get('count');
-		const updatedPullRequests = await loadPullRequests(context.globalState.get('token'), { mode, showMerged, showClosed, repository, showError, count });
+		// configure the URL settings
+		let url = vscode.workspace.getConfiguration('pullRequestMonitor').get('githubEnterpriseUrl');
+		if (url) url = `${url}/api/graphql`;
+		// configure SSL
+		let allowUnsafeSSL = false;
```

Comment:
If you set it as a default `false` in the extension configuration there is no need for all these checks. Also the logic is a bit confusing: if the setting is `null` you pass it on to the function where it defaults `false`. If it's either `true` or `false` you invert a vscode setting which might again be `true` or `false`. Shouldn't you just pass the extension setting?

Reply:
That's true I'll clean it up a bit